code/modules/research/research.dm

```diff
@@ -116,25 +117,31 @@ research holder datum.
 	for(var/datum/design/PD in possible_designs)
 		if(DesignHasReqs(PD))
 			AddDesign2Known(PD)
-	for(var/datum/tech/T in known_tech)
+	for(var/v in known_tech)
+		var/datum/tech/T = known_tech[v]
+		// On tg, this is 1-20, not sure what that is about

```

Comment:
sometimes I whip my weiner around in circles hoping to get it going fast enough to fly away like a helicopter